companydb_select_examples.sql

```diff
@@ -33,13 +33,16 @@ from department;
 
 -- 6. Display the SSN of all employees who live in Houston
 --    (Hint: use LIKE() function as in address LIKE '%Houston'
+select ssn from employee where address LIKE '%Houston%';
```

Comment:
Why %Houston% instead of %Houston?

Reply:
'%Houston' will give the result with address that end with 'Houston' like "638 Fondren, Houston". But it will not give the row that contain 'Houston' in middle of it's address such as "638 Fondren, Houston, TX".
That's why i used '%Houston%' instead of '%Houston'. To consider 'Houston' as a part of address not only the end of address.